index.js

```diff
@@ -0,0 +1,23 @@
+//*  2020 - 03 - 19 //
+// 03 - 19 - 2020 //
+
+let oldDate = "2020 - 03 - 19";
+
+const date = oldDate.split(" ");
+
+let temp = date[0];
```

Comment:
Indices would be wrong because of the offset created by the spaces.
ex
```
with spaces:
["2020, " " , "-" , " ",08" , " ", "-", " ", "02"]
                                     ^
                                      |
                                    index: 4 
without spaces:
["2020, "-" ,08" , "-", "02"]
                                      ^
                                       |
                                    index: 4 
```